modules/pensions/lib/pensions/monitor.rb

```diff
@@ -31,13 +31,17 @@ def track_create_error(in_progress_form, claim, current_user, e = nil)
 
     def track_create_success(in_progress_form, claim, current_user)
       StatsD.increment("#{CLAIM_STATS_KEY}.success")
-      if claim.itf_datetime
-        StatsD.measure('saved_claim.time-to-file', claim.created_at - claim.itf_datetime,
-                       tags: ["form_id:#{claim.form_id}"])
+      if claim.form_start_date
+        claim_duration = claim.created_at - claim.form_start_date # not sure what to call this variable
```

Comment:
Does the variable name `claim_duration` make sense? If so, can you remove this inline comment?